resources/qml/PrintSetupSelector/Recommended/RecommendedQualityProfileSelectorButton.qml

```diff
@@ -0,0 +1,93 @@
+// Copyright (c) 2022 Ultimaker B.V.
+// Cura is released under the terms of the LGPLv3 or higher.
+
+import QtQuick 2.10
+import QtQuick.Controls 2.3
+import QtQuick.Layouts 2.10
+
+import UM 1.5 as UM
+import Cura 1.7 as Cura
+
+
+Rectangle
+{
+    id: base
+    height: 60
+    Layout.fillWidth: true
+    color: mouseArea.containsMouse || selected ? UM.Theme.getColor("um_blue_1") : UM.Theme.getColor("background_1")
```

Comment:
why not use `background_3` instead of `um_blue_1`. It is already in the theme so it has darkmode-support, and then `um_blue_1` can be removed from the themes.json.